### PR TITLE
Commenting and improving code

### PR DESCRIPTION
**Pull request by gwopontes**
Made some improvements in code, like formating, final declarations, removed dead code, unecessarily code and more...

### CODE REVIEW

File: src/main/java/com/totalcross/knowcode/Images.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			dayOnImage = new Image("drawable/day_on.png");
+
+			nightOffImage = new Image("drawable/nigth_off.png");
+			nightOnImage = new Image("drawable/nigth_on.png");
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+
+}
```
Comment (ItaloYeltsin) [inline src/main/java/com/totalcross/knowcode/Images.java:28]:
```suggestion
}

```

File: src/main/java/com/totalcross/knowcode/ReadSensors.java [hunk 2]
```diff
@@ -7,54 +7,75 @@
 import totalcross.sys.Settings;
 import totalcross.sys.Vm;
 
+/**
+ * This is a reader that catch response of dht.py.
+ * 
+ */
 public class ReadSensors {
```
Comment (ItaloYeltsin) [inline src/main/java/com/totalcross/knowcode/ReadSensors.java:14]: I think a more appropriate name for this class would be SensorReader

File: src/main/java/com/totalcross/knowcode/HomeApplianceXML.java [hunk 3]
```diff
@@ -30,249 +26,393 @@
...
 
+	/**
+	 * Queries Firebase Realtime Database for new commands
+	 */
 	private void listen() {
 		try {
+			/*
+			 * Setting http options
+			 */
 			final HttpStream.Options options = new HttpStream.Options();
```
Comment (ItaloYeltsin) [inline src/main/java/com/totalcross/knowcode/HomeApplianceXML.java:388]: not for the pull request, but we could delegate the http calls to other class and reuse some repeated code!

File: src/main/java/com/totalcross/knowcode/ReadSensors.java (hunk 2 above)
Comment (gwopontes) [reply src/main/java/com/totalcross/knowcode/ReadSensors.java:14]: Okay, I'll make it in next commit

**Pull request merged**